cmd/crypto/kv.go

```diff
@@ -0,0 +1,36 @@
+// MinIO Cloud Storage, (C) 2019 MinIO, Inc.
```

Comment:
any reason to follow this style of licensing header while the entire code follows `/* */` style? - can we just use one style?

Reply:
I just copied the license header. Also, std lib style...